sql/report_queries/items_lost_in_transit/items_lost_in_transit_result.csv

```diff
@@ -0,0 +1,55 @@
+date_range,item_id,title,shelving_title,item_status,loan_return_date,checkout_service_point_name,checkin_service_point_name,in_transit_destination_service_point_name,barcode,call_number,enumeration,chronology,copy_number,volume,holdings_permanent_location_name,holdings_temporary_location_name,current_item_permanent_location_name,current_item_temporary_location_name,current_item_effective_location_name,cataloged_date,publication_dates_list,notes_list,material_type_name,num_loans,num_renewals
```

Comment:
Same here, is this supposed to be part of the repository?

Reply:
removed in latest commit